src/server/frontend_wayland/data_device.cpp

```diff
@@ -55,11 +56,12 @@ struct DataOffer : mf::wayland::DataOffer
     DataSource* const source;
 };
 
-struct DataSource : mf::wayland::DataSource
+class DataSource : mf::wayland::DataSource
```

Comment:
What has this to do with this PR? (And why?)

Reply:
err... idk. Seemed like a good idea at the time I s'pose.